Description: fix for removing the ROOT.war directories
  Added -prune to skip dirertory after deleting it.

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- jboss-6.1.0-Final.orig/Makefile
+++ jboss-6.1.0-Final/Makefile
@@ -10,7 +10,7 @@ all:
 	find server -name tmp -exec rm -rf {} \;

   # remove ROOT.war
-	find server -name ROOT.war -exec rm -rf {} \;
+	find server -name ROOT.war -exec rm -rf {} \; -prune

 	true
